### PR TITLE
Add CompatHelper and TagBot + update IRTools bounds

### PR DESCRIPTION
**Pull request by DhairyaLGandhi**
ref #22

### CODE REVIEW

**Comment by DhairyaLGandhi** [comment]
cc @shashi

File: Manifest.toml [hunk 1]
```diff
@@ -9,11 +9,9 @@ uuid = "8ba89e20-285c-5b6f-9357-94700520ee1b"
 
 [[IRTools]]
 deps = ["InteractiveUtils", "MacroTools", "Test"]
-git-tree-sha1 = "5b2ef472bd7001923ea0d703583865ade78c80a3"
-repo-rev = "master"
-repo-url = "https://github.com/MikeInnes/IRTools.jl.git"
+git-tree-sha1 = "a8d88c05a23b44b4da6cf4fb5659e13ff95e0f47"
```
Comment (oxinabox) [inline Manifest.toml:12]: We should just delete the Manifest.toml

File: Project.toml [hunk 2]
```diff
@@ -8,7 +8,6 @@ IRTools = "7869d1d1-7146-5819-86e3-90919afe41df"
 MacroTools = "1914dd2f-81c6-5fcd-8719-6d5c9610ff09"
 
 [compat]
-IRTools = "0.4"
```
Comment (oxinabox) [inline Project.toml:11]: This doesn't seem right

**Review by oxinabox — APPROVED** [review]

**Comment by shashi** [comment]
Awesome!

**Pull request merged**